### PR TITLE
azkv more unittest coverage

### PR DESCRIPTION
**Pull request by dreadful-dragon**
Unit Tests added to have more coverage in the azure Keyvault
Specifically testing to validate the store

### CODE REVIEW

**Comment by moolen** [comment]
Hey @dre, can you refactor this test a bit so the new test cases fit into the table?

**Comment by dreadful-dragon** [comment]
> Hey @dreadful-dragon can you refactor this test a bit so the new test cases fit into the table?

Hello @moolen, what do you mean? Which table?

**Comment by dreadful-dragon** [comment]
> Hey @dreadful-dragon can you refactor this test a bit so the new test cases fit into the table?

Hello @moolen, what do you mean? Which table?

File: pkg/provider/azure/keyvault/keyvault_test.go [hunk 1]
```diff
@@ -483,6 +485,42 @@ func makeValidFind() *esv1beta1.ExternalSecretFind {
...
+		store := getStore(testName)
+		if err := a.ValidateStore(store); (err != nil) != true {
+			t.Errorf(errStore, err, true)
+		}
+	}
+}
+
 func TestValidateStore(t *testing.T) {
 	type args struct {
 		auth esv1beta1.AzureKVAuth
```
Comment (moolen) [inline pkg/provider/azure/keyvault/keyvault_test.go:526]: If you change the type signature here to `*esv1beta1.SecretStore` you can define one individual SecretStore per test case and avoid the complexity

**Review by moolen — CHANGES_REQUESTED** [review]
Hey, sorry for taking such a long time to get back to you :bow:
The test table can be simplified when changing the type, see comment. Can you change that, please? :pray:

**Review by moolen — APPROVED** [review]

**Pull request merged**